src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -965,6 +973,245 @@ void HydrofractureSolver< POROMECHANICS_SOLVER >::setUpDflux_dApertureMatrix( Do
   } );
 }
 
+template< typename POROMECHANICS_SOLVER >
+void HydrofractureSolver< POROMECHANICS_SOLVER >::initializeNewFractureFields( DomainPartition & domain )
+{
+#if !defined(SET_CREATION_DISPLACEMENT)
```

Comment:
can we take advantage of this PR to remove all these macros? We can have two different initialization strategies for now if we are still testing which one works best but let's guard it with  proper flag. 
Also, I don't think we saw that much benefit from initializing the displacement.

Reply:
Fully support to remove the macroses, they are leading to dead code which doesn't even compile